features/utils/apiUtils.js

```diff
@@ -1,14 +1,69 @@
-const axios = require('axios');
+const axios = require("axios");
 
-async function checkUserExists(userId) {
+const apiClient = axios.create({
+  baseURL: "https://reqres.in/api",
+  timeout: 5000,
+});
+
+const getUsersList = async (page) => {
+  try {
```

Comment:
```js
  try {
    if (page) {
     return await apiClient.get(`/users?page=${page}`);
    } 
    return await apiClient.get(`/users`);
  } catch (error) {
    console.error(`Error getting list of users:`, error);
    throw error;
  }
```

Reply:
Not sure about `return await`

Reply:
What is the purpose of this change?

Reply:
Code style and readability. 
Which version is more readable?

Reply:
I find it more readable and convenient to assert when I return response